static/app/stores/drawerStore.tsx

```diff
@@ -0,0 +1,53 @@
+import {createStore} from 'reflux';
```

Comment:
Should we be adding more reflux stores?

Reply:
We don't need reflux for this. This should be a global context.

I want to move modal to be a global context, but I haven't had time.